### PR TITLE
Add shorthand for type

### PR DESCRIPTION
**Pull request by zaaack**
Add shorthand for type, just like other ODMs.

```js
const newSchema = {
  name: String,
  age: Number
}
// equal to
const fullSchema = {
  name: { type: String },
  age: { type: Number }
}

```

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -0,0 +1,1768 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (hiddentao) [inline yarn.lock]: We don't want to have yarn.lock

**Review by hiddentao — COMMENTED** [review]
I actually did want to do it as shorthand originally but I thought that there was a particular reason for not. I think it was - if you had:

``` js
{
  name: {
    type: String
  }
}
```

This says `name` is a `String` or that `name` is an object containing a key `type` which is a `String`. Which one would it be?

File: yarn.lock (hunk 1 above)
Comment (zaaack) [reply yarn.lock]: ok

**Comment by zaaack** [comment]
@hiddentao I think I got some points from you, but I think most people could understand this special case because they have seen enough from other ODMs like  mongoose, came, etc. Think about a big model with lots of unindexed fields (which is common), this could be really useful.

**Comment by hiddentao** [comment]
Sorry for the late reply, busy with other stuff. This can go in but it would be a new major version and the README would need updating.

**Comment by zaaack** [comment]
Thanks for your agreement, recently I figure out what I really need is extensibility, something like passing a function to walk through the configuration of every field and adding custom handling, because I want to add type and index configuration on the schema for mongorito models.

**Pull request merged**